fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -356,6 +358,8 @@ public Response createOrReplaceObjectRdf(
                 }
                 throw new ClientErrorException("Invalid Content Type " + requestContentType, UNSUPPORTED_MEDIA_TYPE);
             }
+        } catch (final Exception e) {
```

Comment:
BinaryStoreException.  But I was concerned that because the exception bubbles up from the modeshape layer,  future versions of modeshape my not sure the BinaryStoreException. Furthermore,  naming the BinaryStoreException would tie create an unnecessary dependency on modeshape/jcr.  There has been talk of trying to decouple jcr from fedora.